in_toto/in_toto_verify.py

```diff
@@ -33,20 +33,28 @@
 from in_toto import verifylib
 from in_toto.models.metadata import Metablock
 
-def in_toto_verify(layout_path, layout_key_paths):
+def in_toto_verify(layout_path, layout_key_paths, layout_gpg_keyids, gpg_home):
   """
   <Purpose>
-    Loads the layout metadata as Metablock object (containg a Layout object)
-    and the signature verification keys from the passed paths,
-    calls   verifylib.in_toto_verify   and handles exceptions.
+    Loads the layout metadata as Metablock object (containing a Layout object)
+    and the signature verification keys from the passed paths and/or from
+    layout_gpg_keyids, calls verifylib.in_toto_verify
```

Comment:
`in_toto_verify.py` can exit with a return value of 0, 1, or 2.  Should you document this in `<Purpose>`?